### ISSUE TITLE
The argument values for the fine-tuning baseline results in mini-ImageNet

### ISSUE THREAD

**Issue by shihux** [cbfinn/maml#26]
Hi Chelsea, could you provide the argument values for the fine-tuning baseline results in mini-ImageNet, as shown in the bottom plot of Table 1? One small problem that I found is when `metatrain_iterations` = 0, because of line 239-241 in main.py, `inputb` becomes an empty tensor (line 269 in main.py). However, `inputb` is used in the function `task_metalearn` in maml.py on line 95.

**Comment by cbfinn** [comment]
The fine-tuning baseline is not implemented in this codebase. The number in the table is from Ravi & Larochelle.

**Comment by shihux** [comment]
Hi Chelsea, thanks for the response. I'm trying to understand the details of the baseline for the mini-imagenet dataset, as the code isn't available. Is it done as follows: 1. Train the same neutral network on all training classes (64 in this case), 2. Keep all the weights in the network but discard the last softmax layer, and then fine tune on the test dataset with k test examples for k shot learning? Thanks

**Comment by cbfinn** [comment]
Yes, that sounds correct. Though, for the details, you would have to ask Sachin.
